test/lender/Lender.sol

```diff
@@ -54,20 +55,17 @@ contract Lender {
     // 'sell base' meaning purchase the zero coupons from yield
     yToken.sellBase(self, returned);
 
-    IMarketPlace mPlace = IMarketPlace(marketPlace);
-    address[8] memory market = mPlace.markets(u, m); 
-
-    uint256 returned256 = Cast.u128(returned);
-
     // this step is only needed when the lend is for yield
     if (p == uint8(MarketPlace.Principals.Yield)) {
+      IMarketPlace mPlace = IMarketPlace(marketPlace);
+      address[8] memory market = mPlace.markets(u, m); 
```

Comment:
this can be inlined now

`address[8] memory market = IMarketPlace(marketPlace).markets(u, m)`